src/screens/home/HomeScreen.js

```diff
@@ -0,0 +1,67 @@
+import React, { useContext, useEffect } from 'react';
+
+// Components
+import { StatusBar } from 'react-native';
+import {
+  Container,
+  Header,
+  Title,
+  Content,
+  Footer,
+  FooterTab,
+  Button,
+  Left,
+  Right,
+  Body,
+  Icon,
+  Text,
+} from 'native-base';
+
+// Context
+import UserContext from '../../contexts/UserContext';
+
+// Vars
+const CONTENT = `This is where any content will go for the home page! I'm not sure what will end up going, but it's going to be good!
+  Use the links at the bottom of the screen to navigate to search or learning pages!`;
+
+const HomeScreen = ({ navigation }) => {
+  const [user] = useContext(UserContext);
+
+  return (
+    <>
+      <StatusBar barStyle="dark-content" />
```

Comment:
Usually we like to create some kind of `ScreenWrapper` component that includes the common bits for each screen (including `StatusBar` and any general app layout stuff which might be nothing else in your case)